src/Platform.Handlers/src/Xamarin.Platform.Handlers/Core/IProgress.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace Xamarin.Platform
+{
+	public interface IProgress : IView
+	{
+		double Progress { get; }
+		Color ProgressColor { get; }
+
+		Task<bool> ProgressTo(double value, uint length, Easing easing);
```

Comment:
"Easing" seems like an animation implementation detail; should that parameter be at the interface level?

Reply:
Right, removed.